winlogbeat/eventlog/eventlogging.go

```diff
@@ -26,10 +26,9 @@ var eventLoggingConfigKeys = append(commonConfigKeys, "ignore_older",
 
 type eventLoggingConfig struct {
 	ConfigCommon     `config:",inline"`
-	IgnoreOlder      time.Duration          `config:"ignore_older"`
-	ReadBufferSize   uint                   `config:"read_buffer_size"   validate:"min=1"`
-	FormatBufferSize uint                   `config:"format_buffer_size" validate:"min=1"`
-	Raw              map[string]interface{} `config:",inline"`
```

Comment:
I guess it used to be for additional 'validation' on allowed settings. But it's not used anymore in any place, so I removed it.